### DISCUSSION TITLE
[Confirmed install platform] Mac Air and Fedora 35

### DISCUSSION THREAD

**Discussion by GChalony** [AppFlowy-IO/AppFlowy#583] [General]
Hi, I just thought I'd share that I successfully installed `AppFlowy` on a MacBook Air with the latest M1 chip, and on a Dell Latitude with Fedora 35.
Maybe that can be added to the doc ?

Thanks for the awesome project anyway :wink:

**Comment by annieappflowy** [comment]
> Hi, I just thought I'd share that I successfully installed `AppFlowy` on a MacBook Air with the latest M1 chip, and on a Dell Latitude with Fedora 35. Maybe that can be added to the doc ?
> 
> Thanks for the awesome project anyway 😉

Thanks @GChalony  for sharing this info. I'll update the doc.

**Comment by visibleLi** [comment]
@GChalony  Hi, can you post the tutorial?  I have the same configuration as you but it doesn't seem to go well。Did you install the app directly or rebuild the image, I look forward to your reply, thank you！

**Comment by GChalony** [comment]
@visibleLi I followed the instructions on the [documentation website](https://appflowy.gitbook.io/docs/essential-documentation/install-appflowy/installation-methods/mac-windows-linux-packages/installing-on-linux). Mac's install is pretty straightforward, linux's is - as usual - a bit more involved :wink:

However I just noticed that I have the same issue as #180 on my linux laptop, but I think that's unrelated.
